gubernator/github/main_test.py

```diff
@@ -60,28 +60,28 @@ def get_response(self, event, body):
             body = json.dumps(body)
         signature = handlers.make_signature(body)
         resp = app.post('/webhook', body,
-            {'X-Github-Event': event,
```

Comment:
Can I have someone verify this won't be an issue? I think case doesn't matter, but I want to verify that.

The spec shows the _corrected_ form: https://developer.github.com/webhooks/#delivery-headers

Reply:
LOL

Reply:
Let's run the tests and see?